source/features/linkify-user-location.tsx

```diff
@@ -32,6 +37,23 @@ function init(): void {
 	}
 }
 
+function searchInit(): void {
+	for (const possibleLocation of select.all('#user_search_results .text-small .mr-3:first-child')) {
```

Comment:
Looking forward to this selector breaking. YOLO

Reply:
First two parts should be reliable enough, I will see how to choose a better last part.